src/engraving/libmscore/key.h

```diff
@@ -61,24 +61,6 @@ enum class Key {
     DELTA_ENHARMONIC = 12
 };
 
-//---------------------------------------------------------
-//   KeyMode
-//---------------------------------------------------------
-
-enum class KeyMode {
-    UNKNOWN = -1,
-    NONE,
-    MAJOR,
-    MINOR,
-    DORIAN,
-    PHRYGIAN,
-    LYDIAN,
-    MIXOLYDIAN,
-    AEOLIAN,
-    IONIAN,
-    LOCRIAN
-};
-
 static inline bool operator<(Key a, Key b) { return static_cast<int>(a) < static_cast<int>(b); }
```

Comment:
I think these operators need to be moved too

Reply:
But these operators are for `Key`, not `KeyMode`? The definition of `Key` itself has not been moved, should it be moved as well?

Reply:
Oh, you're right, I was wrong, no need to move them